Formula/d/dug.rb

```diff
@@ -0,0 +1,23 @@
+class Dug < Formula
+  desc "Global DNS propagation checker that gives pretty output"
+  homepage "https://dug.unfrl.com"
+  url "https://github.com/unfrl/dug/archive/refs/tags/0.0.93.tar.gz"
+  sha256 "a375034bcad3193d47a485b3dd33c7633f7890e5f79b6a9b6b3b3bc198fb2ec3"
+  license "MIT"
+
+  depends_on "dotnet"
+  uses_from_macos "zlib"
+
+  def install
+    ENV["DOTNET_ROOT"] = "$HOMEBREW_PREFIX/opt/dotnet/libexec"
```

Comment:
```suggestion
    ENV["DOTNET_ROOT"] = libexec
```

Reply:
Removed. I think this was just due to trying to debug the usage of dotnet on llinux.

Reply:
sounds good